peanut_gb.h

```diff
@@ -734,12 +779,18 @@ uint8_t __gb_read(struct gb_s *gb, uint16_t addr)
 
 	case 0x8:
 	case 0x9:
+#if PEANUT_FULL_GBC_SUPPORT
+		return gb->vram[addr - gb->cgb.vramBankOffset];
+#else
 		return gb->vram[addr - VRAM_ADDR];
-
+#endif
 	case 0xA:
 	case 0xB:
-		if(gb->mbc == 3 && gb->cart_ram_bank >= 0x08)
+#if PEANUT_FULL_GBC_SUPPORT
+		if(gb->cart_ram && gb->enable_cart_ram)
 		{
+#endif
+		if (gb->mbc == 3 && gb->cart_ram_bank >= 0x08) {
```

Comment:
CGB mode doesn't change the way in which the cart RAM is accessed. This might be a merge conflict not solved correctly.